javascript/javascript3/week3/GitRepos/main.js

```diff
@@ -0,0 +1,30 @@
+let gitRepos = [
+  'https://api.github.com/search/repositories?q=user:claudion3',
+  'https://api.github.com/search/repositories?q=user:Pallabi-ghosh',
+  'https://api.github.com/search/repositories?q=user:nuha90',
+];
+
+let promises = gitRepos.map((url) => {
+  return fetch(url).then((response) => response.json());
+});
+
+Promise.all(promises)
```

Comment:
Good work here also.
Again, I would like to challenge you to think about readability. 
No variables named d, data, element or myhr, please. Noise words.
Inside the then function, I want at least 2 function calls that describe to the reader what you are doing with the responses.